AUTHORS

```diff
@@ -227,22 +234,23 @@ Ubuntu <ubuntu@ip-172-31-15-224.ec2.internal>
 Venkata Krishna Nimmagadda <nvkrishna85@gmail.com>
 Venkata Krishna Nimmagadda <venkata.krishna.nimmagadda@intel.com>
 Vishwas Dsouza <vidsouza@amazon.com>
-Wei Zhang <wzam@amazonc.com>
 Wei Zhang <wzam@amazon.com>
+Wei Zhang <wzam@amazonc.com>
```

Comment:
is this a typo?

Reply:
This was updated based on a script, so the typo likely exists in the commit itself.  I don't know why the script resulted in so many updates to this file, as it's the same script that I alway use.

Reply:
I see, nvm, thanks!